src/App.css

```diff
@@ -20,6 +22,36 @@
   align-items: center;
 }
 
+.nav h1 {
+  color: #0290ff;
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+}
+
+.nav-book {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.nav-cate {
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 41.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
```

Comment:
Kindly implement the following design changes so your design would look better :

- I'd suggest you remove the `author` inbox field from the `Add new Boook` section 👍 
- please use the right margin and padding for the buttons
- kindly add the user icon located at the top right corner of the page
- Kindly reduce the margin of the `Bookstore cms` as shown in the illustration below

![image](https://github.com/maths29/Book-store/assets/20537712/a0c9e44e-6bb8-4481-bbce-25eeb1a2f45c)